wsapi/wsapi.go

```diff
@@ -1571,7 +1571,6 @@ func factoidTxToTransaction(t interfaces.ITransaction) (
 
 	if r.TotalInputs <= r.TotalOutputs+r.TotalECOutputs {
 		r.FeesPaid = 0
-		r.FeesRequired = r.FeesRequired
```

Comment:
I think this line should be something like `r.FeesRequired = feesRequired(t)`, which would properly set the required fee for that field. However, that might be better served as a separate PR from this one since it has potential side effects.